### PR TITLE
Documentation/dev-guide: update "Request size limit" with v3.2.12

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

File: Documentation/dev-guide/limit.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Request size limit
 
-etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. For the time being, etcd guarantees to support RPC requests with up to 1MB of data. In the future, the size limit may be loosened or made configurable.
```
Comment (xiang90) [inline Documentation/dev-guide/limit.md]: the sever side data limit is still 1MB i believe.

Comment (gyuho) [reply Documentation/dev-guide/limit.md]: @xiang90 It's 1.5 MiB now? https://github.com/coreos/etcd/pull/7968

2 MiB was set default 1.5 MiB + gRPC overhead.

File: Documentation/dev-guide/limit.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. As of v3.2.12, etcd guarantees to support client RPC requests of 2 MiB, by default. The client size limit is configurable via `clientv3.Config.MaxCallSendMsgSize`, with corresponding "--max-request-bytes" flag for etcd server.
```
Comment (spzala) [inline Documentation/dev-guide/limit.md]: @gyuho a small observation,  "--max-request-bytes" should change to `--max-request-bytes`. Thanks!

File: Documentation/dev-guide/limit.md (hunk 1 above)
Comment (xiang90) [reply Documentation/dev-guide/limit.md]: Users will not care about what gRPC limits. They care about the size of key+value in put or in transaction. let us put 1.5Mi there, and keep the previous working (not mentioning gRPC client thing here.)

**Comment by gyuho** [comment]
@xiang90 PTAL.

File: Documentation/dev-guide/limit.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## Request size limit
 
-etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. For the time being, etcd guarantees to support RPC requests with up to 1MB of data. In the future, the size limit may be loosened or made configurable.
+etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. As of v3.2.12, etcd guarantees to support client RPC requests of 1.5 MiB, by default. The client size limit is configurable via `clientv3.Config.MaxCallSendMsgSize`, with corresponding `--max-request-bytes` flag for etcd server.
```
Comment (xiang90) [inline Documentation/dev-guide/limit.md]: etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. By default, the maximum size of any request is 1.5 MiB. This limit is configurable through `--max-request-bytes` flag for etcd server.

Comment (xiang90) [reply Documentation/dev-guide/limit.md]: @gyuho if you want to add more documentation on the client side limit. submit a new PR for that and put it into a separate section. i do not want to mix server/client together.

Comment (gyuho) [reply Documentation/dev-guide/limit.md]: Ok sounds good. Will separate out.

**Comment by xiang90** [comment]
lgtm

**Pull request merged**